packit/base_git.py

```diff
@@ -0,0 +1,112 @@
+import git
+import logging
+from typing import Optional
+
+from rebasehelper.specfile import SpecFile
+from packit.config import Config, PackageConfig
+from packit.exceptions import PackitException
+from packit.local_project import LocalProject
+
+logger = logging.getLogger(__name__)
+
+
+class BaseGitClass:
```

Comment:
Can we get a more interesting name? e.g. `class FreddyMercury`

BaseGitClass is not telling me anything.

Reply:
- `PackageGit`
- `PackageRepository`

Reply:
I have selected PackageRepositoryBase.